### PR TITLE
DM-17547: Consolidate obs_lsst README files into one sphinx document

### PR DESCRIPTION
**Pull request by timj**
This pull request merges all the README content into a single sphinx document and adds content on the new metadata translators.

@RobertLuptonTheGood this is mostly your content but I would like you to review my additions.
@SimonKrughoff I took your data/input notes and tests/README and integrated them into the document.

cc/ @jonathansick

### CODE REVIEW

File: doc/lsst.obs.lsst/adding-a-camera.rst [hunk 1]
```diff
@@ -0,0 +1,137 @@
+Adding a new camera
+===================
+
+The ``policy`` directory in the ``obs_lsst`` package contains the files
+needed to describe cameras made up of LSST chips. The eventual goal is
+to describe the real camera, but for now we also have variants to handle
+phosim and imsim simulations (they differ in e.g., the gain and
+crosstalk values).
```
Comment (RobertLuptonTheGood) [inline doc/lsst.obs.lsst/adding-a-camera.rst]: I think we mention at least ts8/comCam/auxTel here too

File: doc/lsst.obs.lsst/adding-a-camera.rst [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+that that’s going to happen).
+
+The basic strategy is that the ``SConscript`` file in the directory
+assembles a suitable ``camera.yaml`` file (e.g. phosim.yaml) and we put
+the appropriate entry in the ``_mapper`` file in the data repository.
+
+To add a new camera (e.g., ``fooCam``, made up of 9 CCDs in a single
+“raft” – call it ``RXX`` for now):
+
+-  Write a header translator for your instrument. This should be placed in
```
Comment (RobertLuptonTheGood) [inline doc/lsst.obs.lsst/adding-a-camera.rst]: I'd rather see the translator discussion moved after how you set up the rafts, as I think that that's how most developers will do things.  E.g. you can visualise the camera with cameraGeom.showCamera before you can read raw files.  To be specific, I'd move the translators _after_ the section about putting a `_mapper` file into your repo.

 Hmm, actually we should mention that now that we have real docs not a README in policy

File: doc/lsst.obs.lsst/adding-a-camera.rst [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+-  Put a ``_mapper`` file in the root of your repository, containing
+   ``lsst.obs.lsstCam.fooCam.FooCamMapper``
+-  Retarget ``config.parse`` in ``config/fooCam/ingest.py`` to
+   ``FooCamParseTask``
+-  You will probably also want to copy e.g., ``config/auxTel/auxTel.py``
+   to ``config/fooCam/fooCam.py`` and also files such as
+   config/auxTel/bias.py – don’t forget to modify them to import
+   ``fooCam.py``!
+-  Add ``FooCam.yaml`` to ``policy/.gitignore``
+-  Generate a raw test data file (or files) that is as small as possible.
```
Comment (RobertLuptonTheGood) [inline doc/lsst.obs.lsst/adding-a-camera.rst]: I'd put this under a separate heading, adding tests.  While this is absolutely necessary, it's separate from getting your camera working; e.g. as written it seems as if these fake calibs are somehow relevant to handling real data.

File: doc/lsst.obs.lsst/adding-a-camera.rst (hunk 2 above)
Comment (timj) [reply doc/lsst.obs.lsst/adding-a-camera.rst]: I will say that I always write the translators before I even think about the camera definition or Mapper but I'll move the order.

**Pull request merged**